### PR TITLE
Consuming unsplash API using Node.js graphql API

### PR DESCRIPTION
**Pull request by mwangiKibui**
### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by mwangiKibui** [comment]
This fixes #1445

**Comment by katungi** [comment]
@hectorkambow I would also like to review this article

**Comment by katungi** [comment]
Hey there @mwangiKibui This is an amazing article and will be of help to a lot of developers.

File: articles/consuming-unsplash-api-using-node.js-graphql-api/index.md [hunk 1]
```diff
@@ -0,0 +1,436 @@
+Unsplash is a photography sharing application. It has millions of users who post photos related to different topics every day. It is estimated to have more than two million photos. With the need to bring their services closer to developers, Unsplash released their developer API. With over six billion requests per month, it provides an infrastructure for developers to build an experience for their users using services provided by Unsplash. In this article, we will walk through the steps of creating an Unsplash developer account and implement various functionalities of the API on a Graphql API.
```
Comment (katungi) [inline articles/consuming-unsplash-api-using-node.js-graphql-api/index.md:1]: It is general good practice to add a title to the article.

**Review by katungi — COMMENTED** [review]
Just a few notes:
1. Add a title to the document
2. After writing a title, remove the trailing punctuation and spaces

File: articles/consuming-unsplash-api-using-node.js-graphql-api/index.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+
+#### prerequisites.
```
Comment (katungi) [inline articles/consuming-unsplash-api-using-node.js-graphql-api/index.md:3]: remove the full stop. This applies to all the titles.

**Comment by katungi** [comment]
Everything looks fine, just the few formatting changes. Finished the preliminary review, moving forward to the tech review!

**Comment by katungi** [comment]
@mwangiKibui I have set up the project just as the article explains. When I run my queries however, I keep getting an error. I then tried the same with the final code and I get the same error, the api returns null. Can you check on that please?

PS: I created a developer account and added my access key accordingly.

File: articles/consuming-unsplash-api-using-node.js-graphql-api/index.md (hunk 1 above)
Comment (mwangiKibui) [reply articles/consuming-unsplash-api-using-node.js-graphql-api/index.md:3]: fixed.

File: articles/consuming-unsplash-api-using-node.js-graphql-api/index.md (hunk 2 above)
Comment (mwangiKibui) [reply articles/consuming-unsplash-api-using-node.js-graphql-api/index.md]: fixed.

**Comment by mwangiKibui** [comment]
@katungi I have solved the technical error you had pointed out. Everything works nice on my side. However, if the error still persists on your side feel free to inform with a screenshot. Thanks.

**Comment by katungi** [comment]
> @katungi I have solved the technical error you had pointed out. Everything works nice on my side. However, if the error still persists on your side feel free to inform with a screenshot. Thanks.

Have you pushed the changes to the demo repo so I can pull the changes?

**Comment by mwangiKibui** [comment]
@katungi yes I have. Check it out.

**Comment by katungi** [comment]
@mwangiKibui The code now works and one can query the data well.
@hectorkambow, I am done with the tech review and everything looks good to me.

**Comment by mwangiKibui** [comment]
@katungi Okay. Thanks.

**Pull request merged**